changehc/delphi_changehc/load_data.py

```diff
@@ -104,11 +107,16 @@ def load_combined_data(denom_filepath, covid_filepath, dropdate, base_geo):
     data["den"] = data[Config.DENOM_COL]
     data = data[["num", "den"]]
 
+    # Store for backfill
+    merge_backfill_file(backfill_dir, numtype, geo, weekday, backfill_merge_day,
+                        dropdate, test_mode=False, check_nd=25)
+    store_backfill_file(data, dropdate, backfill_dir, numtype, geo, weekday)
```

Comment:
I want to make it to be consistent with the checking criterial. Together with the problem that you mentioned above. If we set a looser threshold because of the `earliest date`, then we have a strict cut off here to merge the existing files before we save new backfill files. If we allow the flexibility for `today`, then we would store the files before merging them.